src/runner/browser-env/vite/plugins/generate-index-html.ts

```diff
@@ -59,21 +60,17 @@ export const plugin = async (): Promise<Plugin[]> => {
                         }
 
                         const urlParsed = url.parse(req.originalUrl);
-                        const urlParamString = new URLSearchParams(urlParsed.query || "");
+                        const runUuid = _.compact(urlParsed.pathname?.split("/"))[0];
```

Comment:
And here we could look for `run-uuid-\d+` path component, instead of just taking the first one